stdlib/src/builtin/simd.mojo

```diff
@@ -1620,17 +1620,8 @@ struct SIMD[type: DType, size: Int = simdwidthof[type]()](
             `(self+other)[permutation[i]]`.
         """
 
-        @parameter
```

Comment:
I didn't realize that I deleted this check, but I cannot get it to work, keeping ` <= mask[i] < 2 * size` gives the following error: 
```
no matching method in call to '__refitem__'mojo
static_tuple.mojo(341, 8): candidate not viable: callee expects at least 2 positional parameters, but 0 were specified
static_tuple.mojo(372, 8): candidate not viable: expected at most 1 positional argument, got 2
```

Reply:
I saw that the reference changed a lot in the last nightly release so after updating the error changed to `cannot use a dynamic value in call parameter` which makes a lot more sense, but I'm still not sure how to add the compile time check again

Reply:
Ok so I dig this one a bit and it's a compiler bug. The good news is that I found a workaround, as the struct `Tuple` had the same problem (I was wondering how Tuple handled this). So I found this: https://github.com/modularml/mojo/blob/nightly/stdlib/src/builtin/tuple.mojo#L182 I used the same trick here and it worked. It seems the issue is tracked internally, so this problem should go away at some point.